### DISCUSSION TITLE
[Support] 如何解析POST请求Content-Type为text/html的内容？

### DISCUSSION THREAD

**Discussion by elicip** [midwayjs/midway#4029] [Q&A] [answered]
### Describe the problem（描述问题）

因为要对接一个二维码设备，他们发送的数据格式是text/html，我使用ctx.request.rawBody 或者ctx.request.body都无法获取内容，midwayjs的解析器无法解析text/html吗？

### Midway Versions（Midway 版本）

npx midway-version@latest

**Comment by waitingsong** [comment]
POST 表单？

**Reply by czy88840616** [reply, marked as answer]
不解析非标类型，这个需要自行处理。
